.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -1,13 +1,13 @@
 name: Bug Report
 description: File a bug report
-labels: [kind: bug]
+labels: "kind: bug"
```

Comment:
Looks like we need apostrophes here, as the current version doesn't create any labels.